stock_warehouse_calendar_orderpoint/models/stock_warehouse_orderpoint.py

```diff
@@ -0,0 +1,62 @@
+# Copyright 2022 Camptocamp SA
+# License LGPL-3.0 or later (http://www.gnu.org/licenses/lgpl)
+
+
+from dateutil import relativedelta
+
+from odoo import api, fields, models
+
+
+class StockWarehouseOrderpoint(models.Model):
+    _inherit = "stock.warehouse.orderpoint"
+
+    @api.depends(
+        "rule_ids",
+        "product_id.seller_ids",
+        "product_id.seller_ids.delay",
+        "warehouse_id.orderpoint_calendar_id",
+    )
+    def _compute_lead_days(self):
```

Comment:
It seems to me this code should be implemented in `stock_warehouse_calendar`, as it would be the same scope only the original implementation maybe forgot about the lead_days computation and its impact in the created purchase orders from orderpoints.

I would love to see a hook method to retrieve the calendar_id here.
(thinking of an usecase we have, to have calendars set on suppliers)